### PR TITLE
Tag version 1.1.0

### PR DESCRIPTION
**Pull request by bitjammer**

### CODE REVIEW

**Comment by bitjammer** [comment]
@swift-ci Please test

**Comment by bitjammer** [comment]
(One can never be too sure)

**Comment by bitjammer** [comment]
Added copyright to the changelog.

**Comment by bitjammer** [comment]
@swift-ci Please test

**Comment by bitjammer** [comment]
Moved it to the top.

**Comment by bitjammer** [comment]
@swift-ci Please test

**Comment by bitjammer** [comment]
Whitespace.

**Comment by bitjammer** [comment]
@swift-ci Please test

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# CHANGELOG
+
+<!-- Add items in reverse chronological order. -->
+
+<!-- Add new items to an **## Unreleased** section as needed. -->
+
+## 1.1.0
+
+- Adds support for [snippets](https://github.com/apple/swift-evolution/blob/main/proposals/0356-swift-snippets.md).
```
Comment (ethan-kusters) [inline CHANGELOG.md:9]: Should we mention here the `snippets` support is limited to Swift 5.7 or would you rather leave that to the proposal?

**Review by ethan-kusters — APPROVED** [review]
Thank you!

File: CHANGELOG.md (hunk 1 above)
Comment (bitjammer) [reply CHANGELOG.md:9]: I think it's fine to leave it as is.

**Pull request merged**